build-support/bin/generate_travis_yml.py

```diff
@@ -225,13 +225,20 @@ def __str__(self) -> str:
     return str(self.value)
 
 
-def _linux_before_install(include_test_config: bool = True) -> List[str]:
+def _linux_before_install(include_test_config: bool = True, travis_wait: bool = False) -> List[str]:
   commands = [
     "./build-support/bin/install_aws_cli_for_ci.sh",
     # TODO(John Sirois): Get rid of this in favor of explicitly adding pyenv versions to the PATH:
     #   https://github.com/pantsbuild/pants/issues/7601
-    "pyenv global 2.7.15 3.6.7 3.7.1"
+    "pyenv global 2.7.15 3.6.7 3.7.1",
   ]
+  if travis_wait:
+    commands += [
+      'wget -qO- "https://github.com/crazy-max/travis-wait-enhanced/releases/download/v0.2.1/travis-wait-enhanced_0.2'
+      '.1_linux_x86_64.tar.gz" | tar -zxvf - travis-wait-enhanced',
+      "mv travis-wait-enhanced /home/travis/bin/",
+    ]
```

Comment:
Nit: preferred style is `commands.extend([...])`

Reply:
Also this is very easy to miss that the first two lines are concatenated and last is a new entry. This is ugly to do, but I recommend using parentheses to make this explicit:

```
     [
       (
          'wget -qO- "https://github.com/crazy-max/travis-wait-enhanced/releases/download/v0.2.1/travis-wait-enhanced_0.2'
          '.1_linux_x86_64.tar.gz" | tar -zxvf - travis-wait-enhanced'
        ),
        "mv travis-wait-enhanced /home/travis/bin/",
      ]

Reply:
was just following the lead here :) https://github.com/pantsbuild/pants/pull/8446/files/98123e1d74e0865dacb3eef8cf2efa03769a5ed7#diff-b1e05cd261dcac1b57c585024101ca65R247

Happy to change one or both.

Reply:
I'm not sure what that links to, but feel free to change the original too! One way to verify it works: `.travis.yml` should not be impacted at all.

Reply:
oh thats super annoying that the link doesn't work. One of the lines below uses list addition to add commands to the list.